.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 *       @mongodb/mongocli
 
 # Docs maintained by Docs Cloud Team
-docs/*  @mongodb/docs-cloud-team
+/docs/  @mongodb/docs-cloud-team
```

Comment:
@fmenezes 
```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
```
https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners